project/plugin.sbt

```diff
@@ -1 +1,2 @@
 scalacOptions ++= Seq("-unchecked", "-deprecation")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.5")
```

Comment:
Can you move the plugin to separate file called 'sbt-scalafix.sbt' (like 'sbt-sonatype.sbt') and add comment with the link to source  (like in 'sbt-sonatype.sbt')